book/07-git-tools/sections/revision-selection.asc

```diff
@@ -82,8 +82,13 @@ That's 1,200 times the number of grains of sand on the earth.
 
 Here's an example to give you an idea of what it would take to get a SHA-1 collision.
 If all 6.5 billion humans on Earth were programming, and every second, each one was producing code that was the equivalent of the entire Linux kernel history (6.5 million Git objects) and pushing it into one enormous Git repository, it would take roughly 2 years until that repository contained enough objects to have a 50% probability of a single SHA-1 object collision.
-Thus, a SHA-1 collision is less likely than every member of your programming team being attacked and killed by wolves in unrelated incidents on the same night.
+Thus, an organic SHA-1 collision is less likely than every member of your programming team being attacked and killed by wolves in unrelated incidents on the same night.
 
+[NOTE]
+====
+There have been attempts to create a synthetic collision attack, including one documented at https://shattered.io/[].
+Git is moving towards using SHA256 as the default hashing algorithm, which is much more resilient to collision attacks, and has code in place to help mitigate this attack (although it cannot completely eliminate it).
+====
 ====
```

Comment:
Oof, just now realizing that we're already inside a note. Let's remove the nesting, I don't think it'll work:

```suggestion
There have been attempts to create a synthetic collision attack, including one documented at https://shattered.io/[].
Git is moving towards using SHA256 as the default hashing algorithm, which is much more resilient to collision attacks, and has code in place to help mitigate this attack (although it cannot completely eliminate it).
====
```